docs/how-to-guides/how-to-config-layers.md

```diff
@@ -0,0 +1,76 @@
+# How to Config Layers
+
+Below is an example of the current configuration format. For full details of all fields, see the [complete layer specification](../reference/layer-specification).
+
+```yaml
+summary: Simple layer
+
+description: |
+    A better description for a simple layer.
+
+services:
+    srv1:
+        override: replace
+        summary: Service summary
+        command: cmd arg1 "arg2a arg2b"
+        startup: enabled
+        after:
+            - srv2
+        before:
+            - srv3
+        requires:
+            - srv2
+            - srv3
+        environment:
+            VAR1: val1
+            VAR2: val2
+            VAR3: val3
+
+    srv2:
+        override: replace
+        startup: enabled
+        command: cmd
+        before:
+            - srv3
+
+    srv3:
+        override: replace
+        command: cmd
+```
+
+The `override` field (which is required) defines whether this entry _overrides_ the previous service of the same name (if any), or merges with it. See the [full layer specification](../reference/layer-specification) for more details.
+
+---
```

Comment:
I do like how this looks, but I don't think we should go adding these between every section. If anything, that should be done in the styling/CSS (and probably at the Canonical / starer pack level if we're going to do it).